src/briefcase/app.py

```diff
@@ -280,7 +283,8 @@ def _git_checkout(self, path):
             subprocess.check_output(["git", "checkout", self._python_version], stderr=subprocess.STDOUT, cwd=path)
         except subprocess.CalledProcessError as pull_error:
             error_message = pull_error.output.decode('utf-8')
```

Comment:
At column 13: (F841) local variable 'error_message' is assigned to but never used

Reply:
I didn't touch this code though - this should've been spotted before it was in master!

Reply:
This has nothing to do with this PR. I guess I'll fix it anyway